### PR TITLE
feat: add text-embedding in dashscope

### PR DESCRIPTION
**Pull request by Abirdcfly**
For #101 
full output 
[output.txt](https://github.com/kubeagi/arcadia/files/12855733/output.txt)
for embedding:
CleanShot 2023-10-10 at 17 58 36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ To enhace the AI capability in Golang, we developed some packages.
 
 - ✅ [ZhiPuAI(智谱 AI)](https://github.com/kubeagi/arcadia/tree/main/pkg/llms/zhipuai)
   - [example](https://github.com/kubeagi/arcadia/blob/main/examples/zhipuai/main.go)
+- ✅ [DashScope(灵积模型服务)](https://github.com/kubeagi/arcadia/tree/main/pkg/llms/dashscope), we now support
```
Comment (bjwswang) [inline README.md:65]: @Abirdcfly pls also update the model list in zhipuai .

File: pkg/llms/dashscope/api.go [hunk 2]
```diff
@@ -62,9 +68,105 @@ func (z *DashScope) Call(data []byte) (llms.Response, error) {
...
+	}
+	return respData.Output.Embeddings, nil
+}
+
+func (z *DashScope) CreateEmbeddingAsync(ctx context.Context, inputURL string, query bool) (taskID string, err error) {
+	textType := TextTypeDocument
+	if query {
+		textType = TextTypeQuery
+	}
+	reqBody := EmbeddingRequest{
```
Comment (bjwswang) [inline pkg/llms/dashscope/api.go:118]: The async mode only allows url as the input.Right?

**Comment by bjwswang** [comment]
@Abirdcfly once this pr merged,pls update llm's controller to support dashscope

File: pkg/llms/dashscope/api.go (hunk 2 above)
Comment (Abirdcfly) [reply pkg/llms/dashscope/api.go:118]: yes, async -> url / sync-> text.

**Review by bjwswang — APPROVED** [review]

**Pull request merged**